src/components/eci/Captcha.js

```diff
@@ -52,7 +52,8 @@ export default function Captcha(props) {
   const [audioCaptcha, _setAudioCaptcha] = useState(false);
   const withAudioCaptcha = config.component.captcha?.audio !== false;
   const { t } = useTranslation();
-  const { setValue, errors } = props.form;
+
+  const { setValue, formState: { errors } } = props.form; // in react-hook-form 7, errors are stored in formState
```

Comment:
@1v4n4 did you test for an eci if everything works fine?

Reply:
It is hard to tell. I tested whatever I could think of, but there are so many options...

Reply:
we need more unit tests indeed

here: test if you can sign without a captcha (no), if you can sign with an invalid captcha (no), and can sign with a real one (yes)

Reply:
captcha already tested for nonexisting and wrong and it works great